test/e2e/scenarios/add_cluster.go

```diff
@@ -0,0 +1,18 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Adding existng network policy to newly added cluster", func() {
+	PIt("Should implement existng network policy in newly added cluster", func() {
+		By("creating pod1 with label1 in cluster1 in namespace1")
+		By("creating pod2 with label2 in cluster2 in namespace1")
+		By("creating network policy with namespace selector")
+		By("testing connectivity between pods")
+		By("Create a new cluster")
+		By("Discover the newly created cluster")
+		By("creating pod3 with label3 in cluster3 in namespace3")
```

Comment:
We also need the other way around:

1) Pod added to cluster 3 on namespace 3
2) Registering the cluster..